tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -532,18 +560,41 @@ private void PostIteration(ScenarioExecutionResult scenarioExecutionResult)
                     firstRequest.Performance.Metrics.Add(ElapsedTimeMilliseconds);
                 }
 
-                startup.Performance.IterationModels.Add(new IterationModel {
+                var medianResponse = scenarioBenchmark.Tests
+                    .SingleOrDefault(t => t.Name == "Median Response" && t.Namespace == scenarioTestModel.Name);
```

Comment:
I think having "Median" as a test name does not feel right. Currently, we only gather the time for a response, but we might gather more data on the future.

Reply:
So maybe "Median Response Time"?

Reply:
I was thinking "Response" only. "Time" is the metric captured for the test, and "Median" might be confusing/redundant here. On the site, we report different aggregation for the captured values (Average, Median, Best Value, Min, Max, etc.)

Reply:
The confusing thing is that there are two levels of aggregation here -- the app is reporting the median response time from a series of 1001 requests, and the post-processing is then aggregating the data from N of these runs and reporting a single metric, where as you note the aggregation method can vary.

The app also still reports the average response time for the 1001 requests, but I find it a less desirable metric because of outliers.

So it feels to me like "median" ought to be in there, but I don't care that strongly.

Reply:
OK, it makes sense. We can leave it as-is.